packages/manager/modules/product-offers/src/workflows/product-offers-order-workflow.class.js

```diff
@@ -165,7 +165,9 @@ export default class OrderWorkflow extends Workflow {
       configuration,
     };
 
-    const serviceName = this.serviceNameToAddProduct;
+    const serviceName = isFunction(this.serviceNameToAddProduct)
```

Comment:
By using getter, normally you can just keep `const serviceName = this.serviceNameToAddProduct;`

Reply:
In fact i need to have something to dynamic. Because in my order funnel database the all type of database haven't same product name and one need service name because is an option other no because is individual service.